widget/playground/src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -60,7 +60,7 @@ export function MultiSelect(props: MuliSelectPropTypes) {
               <Chip label={`+${value.length - MAX_CHIPS}`} variant="outlined" />
             )}
           </div>
-          <ChevronRightIcon size={12} />
+          <ChevronRightIcon size={12} color="black" />
```

Comment:
Based on Figma I think it should be gray.

Reply:
fixed